tools/azure-devtools/src/azure_devtools/scenario_tests/preparers.py

```diff
@@ -145,13 +145,14 @@ def _preparer_wrapper(test_class_instance, **kwargs):
             try:
                 try:
                     import asyncio
+                except ImportError:
```

Comment:
I don't understand the prior comment--I'm thinking we don't need to handle `SyntaxError` because shouldn't this always be `ImportError` on 2.7 (modulo someone's custom `asyncio` module)?

Reply:
I concur